### PR TITLE
Pre-selected text in the search bar

### PR DESCRIPTION
**Pull request by sarvanideekshitula**
**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [ ] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: 
fixes: #88

### CODE REVIEW

**Comment by AdiChat** [comment]
The text within the search bar is selected always which looks odd. A better option will be to add a clear button in form of a cross in the search bar. Also, add a search button in case, the user does not want to press enter. This will be an effective and quick change. 

Kindly take a look at it. :+1:

File: search/templates/cosmos/header.html [hunk 1]
```diff
@@ -10,6 +10,9 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
     <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (AdiChat) [inline search/templates/cosmos/header.html]: It looks like fontawesome is already included. Kindly take a look into it. :+1:

File: search/templates/cosmos/header.html [hunk 2]
```diff
@@ -10,6 +10,9 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
     <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (AdiChat) [inline search/templates/cosmos/header.html]: You must include scripts at the end of the file. :+1:

**Comment by AdiChat** [comment]
I have added a few comments. Kindly take a look at it. :+1:

**Comment by sarvanideekshitula** [comment]
@AdiChat Please review

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**